miden/tests/integration/main.rs

```diff
@@ -28,15 +28,15 @@ fn multi_output_program() {
 /// specified stack and advice inputs.
 ///
 /// Parameters are expected in the following order:
-/// `source`, `stack_inputs` (optional), `advice_tape` (optional), `merkle_sets` (optional)
+/// `source`, `stack_inputs` (optional), `advice_stack` (optional), `merkle_sets` (optional)
 ///
 /// * `source`: a string of one or more operations, e.g. "push.1 push.2".
 /// * `stack_inputs` (optional): the initial inputs which must be at the top of the stack before
 /// executing the `source`. Stack inputs can be provided independently without any advice inputs.
-/// * `advice_tape` (optional): the initial advice tape values. When provided, `stack_inputs` and
+/// * `advice_stack` (optional): the initial advice stack values. When provided, `stack_inputs` and
 /// `merkle_sets` are also expected.
 /// * `merkle_sets` (optional): the initial merkle set values. When provided, `stack_inputs` and
-/// `advice_tape` are also expected.
+/// `advice_stack` are also expected.
```

Comment:
We should probably change `merkle_sets` to `merkle_store` here and in other macros in this file.

Reply:
Done!